src/SourceBuild/patches/nuget-client/0001-Remove-auditSources-section.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Tue, 13 Aug 2024 18:57:27 +0000
+Subject: [PATCH] Remove auditSources section
+
+Tracking: https://github.com/dotnet/source-build/issues/4548
```

Comment:
For future reference the convention is `Backport:` as documented in the [patch guidance](https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines).  Also the referenced issue/PR should track the work to remove the patch.  This issue was just closed by this PR.  What issue is tracking the long term fix?